README.md

```diff
@@ -109,26 +116,26 @@ This always compiles the latest nightly builds. If you want to build stable, do
 $ git checkout stable
 ```
 
-### 3.3 Starting Open Ethereum <a id="chapter-0034"></a>
+### 3.3 Starting OpenEthereum <a id="chapter-0034"></a>
 
 #### Manually
 
-To start Open Ethereum manually, just run
+To start OpenEthereum manually, just run
 
 ```bash
 $ ./target/release/parity
 ```
 
-so Open Ethereum begins syncing the Ethereum blockchain.
+so OpenEthereum begins syncing the Ethereum blockchain.
 
 #### Using `systemd` service file
 
-To start Open Ethereum as a regular user using `systemd` init:
+To start OpenEthereum as a regular user using `systemd` init:
 
 1. Copy `./scripts/parity.service` to your
 `systemd` user directory (usually `~/.config/systemd/user`).
 2. Copy release to bin folder, write `sudo install ./target/release/parity /usr/bin/parity`
-3. To configure Open Ethereum, write a `/etc/parity/config.toml` config file, see [our old wiki](https://paritytech.github.io/wiki/Configuring-Parity) for details.
+3. To configure OpenEthereum, write a `/etc/parity/config.toml` config file, see [our old wiki](https://paritytech.github.io/wiki/Configuring-Parity) for details.
```

Comment:
```suggestion
3. To configure OpenEthereum, write a `/etc/openethereum/config.toml` config file, see [our old wiki](https://paritytech.github.io/wiki/Configuring-Parity) for details.
```

I don't think that path is special but I'm not 100% sure!

Reply:
Yes, it is not used unless `--config` is used

Reply:
If it's not used, then we should remove this point?

Reply:
I was wrong, it is used if the `BASE_PATH` is `/etc/openethereum` and can be overwritten by `--config ..../some/some.toml` but I think the following with be sufficent:

> To configure OpenEthereum, see [our old wiki](https://paritytech.github.io/wiki/Configuring-Parity) for details.